core/src/main/java/org/apache/carbondata/core/metadata/schema/table/CarbonTable.java

```diff
@@ -218,6 +222,41 @@ public static void updateTableInfo(TableInfo tableInfo) {
     }
   }
 
+  public static CarbonTable buildDummyTable(
```

Comment:
I think it is not a dummy table. it is actual table.

Reply:
no need table name and db name to build table info when non filter